packages/html-reporter/src/metadataView.tsx

```diff
@@ -82,32 +82,27 @@ const InnerMetadataView: React.FC<{ metadata: Metadata }> = params => {
 };
 
 const CiInfoView: React.FC<{ info: CIInfo }> = ({ info }) => {
-  const link = info.commitHref;
+  const title = info.prTitle || `Commit #${info.commitHash}`;
```

Comment:
```suggestion
  const title = info.prTitle || `Commit ${info.commitHash}`;
```
I don't think we want `#17c4d8e`